app/policies/trainee_policy.rb

```diff
@@ -107,6 +107,14 @@ def destroy_with_reason?
   alias_method :confirm?, :update?
   alias_method :delete?, :destroy?
 
+  def write_placements?
```

Comment:
Maybe something for the future, I think it would have been good to adjust the original `write?` method to handle the new logic. We should probably avoid having feature specific permissions but try to remove the restriction at a category-like level of features incase we need to apply this logic to others in the future. `awaiting_action?` sounds like it could be adjusted to handle awarded states but still waiting on information like placements

Reply:
Agree that `write_placements?` doesn't fit well. I did wonder whether we would be better off with a `PlacementPolicy` (with a simple `write?` method) and discussed it with @carlosmartinez The main issue with that is that we wouldn't have a `Placement` instance, only a trainee. Since all the authorisation logic depends on the trainee (rather than an individual placement) it may make sense.